packages/mdc-top-app-bar/_mixins.scss

```diff
@@ -80,6 +80,16 @@
         @include mdc-rtl-reflexive-box(padding, right, $mdc-top-app-bar-short-collapsed-right-icon-padding);
       }
     }
+
+    .mdc-top-app-bar--prominent {
+      .mdc-top-app-bar__row {
+        height: $mdc-top-app-bar-prominent-row-height;
```

Comment:
Also, shouldn't this line not even be necessary if you decrease the specificity of `.mdc-top-app-bar .mdc-top-app-bar__row` to just the latter class, further above in this mixin? I don't think those first 3 selectors should need 2 classes each, since they don't have 2 classes each in the default styles.

Reply:
The first three selectors do have 2 classes each in the default styles.